File: BaekJoon/test1/Three_num.c
#include<stdio.h>
int main(){
    int input[3];
    int temp;
    scanf("%d %d %d",&input[0],&input[1],&input[2]);
    
    for(int i=0;i<3; i++){
        for(int j=0; j<2; j++){
            if(input[j]>= input[j+1]){
                temp = input[j];
                input[j]=input[j+1];
                input[j+1]=temp;
            }
        }
    }
    printf("%d\n", input[1]);
    return 0;
}

File: BaekJoon/test1/three_num.js
const readline = require("readline");
const rl = readline.createInterface({
    input: process.stdin,
    output : process.stdout
});

let input=[];

rl.on('line',function(line){
    input=line.split(' ').map((el)=>parseInt(el));
})
.on('close', function( ){
    let numbers = [];
    //let temp;
    for(var i=0; i<input.length;i++){
        numbers.push(Number(input[i]));
    }

    numbers.sort();

    console.log(numbers[1]);

    process.exit();
});
